scripts/shell.sh

```diff
@@ -125,16 +128,17 @@ setup_nix() {
   }
   $NIX_USER_CHROOT_BIN "${NIX_STORE}" sh -c\
    "curl -L https://releases.nixos.org/nix/nix-${NIX_VERSION}/install > ${NIX_INSTALL_SCRIPT} && sh ${NIX_INSTALL_SCRIPT}\
+    --no-channel-add\
     --no-daemon\
     --no-modify-profile\
-    --nix-extra-conf-file ${NIX_EXTRA_CONF_PATH}"
+    --nix-extra-conf-file ${NIX_CONF_DIR}/nix.conf"
 }
 
 ensure_direnv_is_configured() {
-  local readonly project_root=$(dirname $(realpath ${__FILE__}/../docs))
+  local readonly project_root=$(dirname $(realpath ${__DIR__}/../docs))
```

Comment:
Hmm, why does it point to the `docs` and not to the root of project itself? :thinking:

Reply:
Good catch - that was leftover from my tinkering. Simplified.